### PR TITLE
#246 Remove extra call to vacation response when selecting a mailbox

### PR DESCRIPTION
**Pull request by oubchid**
1. Removing duplicate call for **getVacationResponses** 
Already called here [vacation-banner.controller.js](https://github.com/OpenPaaS-Suite/esn-frontend-inbox/blob/2d36f27777490dddb960e7ce54097022a086df76/src/linagora.esn.unifiedinbox/app/components/banner/vacation-banner/vacation-banner.controller.js#L14)

### CODE REVIEW

**Review by rezk2ll — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

**Review by rezk2ll — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Comment by chibenwa** [comment]
Please avoid to dismiss people reviews, that prevents next reviewers to see previous comments.

You might also by doing so erase valuable discussions.

My 2 cents.

**Comment by rezk2ll** [comment]
> Please avoid to dismiss people reviews, that prevents next reviewers to see previous comments.
> 
> You might also by doing so erase valuable discussions.
> 
> My 2 cents.

the dismiss review part is automatic, what happens is:
1- the branch is out of date compared to the `main` branch. ( happens a lot cause other people tend to merge their PRs while others wait for reviews )
2- people approve the pull request.
3- developer can't merge his pull request because of (1) and is forced to rebase his branch and push again.
4- GitHub automatically removes all the approvals since there was a forced push.
5- we have to ask people to reapprove again.

we can use use the update branch button as Michael does ( but this adds another commit just for updating the PR )

**Comment by oubchid** [comment]
> > Please avoid to dismiss people reviews, that prevents next reviewers to see previous comments.
> > You might also by doing so erase valuable discussions.
> > My 2 cents.
> 
> the dismiss review part is automatic, what happens is:
> 1- the branch is out of date compared to the `main` branch. ( happens a lot cause other people tend to merge their PRs while others wait for reviews )
> 2- people approve the pull request.
> 3- developer can't merge his pull request because of (1) and is forced to rebase his branch and push again.
> 4- GitHub automatically removes all the approvals since there was a forced push.
> 5- we have to ask people to reapprove again.
> 
> we can use use the update branch button as Michael does to not lose the approvals ( but this adds another commit just for updating the PR )

Yes right it's automatic, and we should wait 24 hours before merge, to permit to all others to review

**Pull request merged**
